packages/apps/docs/src/components/Layout/components/TitleHeader/style.css.ts

```diff
@@ -73,7 +73,7 @@ export const headerClass = style([
   {
     gridArea: 'main',
     zIndex: 3,
-    backgroundColor: $$backgroundOverlayColor,
+    backgroundColor: 'rgba(250,250,250, .8)',
```

Comment:
the variable is already this color

Reply:
this will not work in darkmode

Reply:
ah okay! But actually we can't use this same global variable import from this file because the context has changed now after moving TileHeader out of the Main content area. But I'll have a look and define it in variable.